### PR TITLE
Add validation to verify the blacklisted drivers 

### PR DESCRIPTION
**Pull request by anujkmaurya**
PBI:

1.   https://msazure.visualstudio.com/One/_workitems/edit/13135418
2. https://msazure.visualstudio.com/One/_workitems/edit/12448967

Added validations to check blacklisted drivers in the modprobe configuration.

### CODE REVIEW

**Comment by anujkmaurya** [comment]
task :https://msazure.visualstudio.com/One/_workitems/edit/12607531

File: ansible_image_validation/validate-vm-images.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         err_folder: /tmp/err
         license_type: "{{ 'byos' if 'byos' in inventory_hostname else 'payg' }}"
         repo_type: "{{'base' if 'base' in offer_type else ('byol' if 'byol' in offer_type else 'eus')}}"
```
Comment (simratsatia) [inline ansible_image_validation/validate-vm-images.yaml:25]: I don't see any changes required in this file. Please revert them

File: ansible_image_validation/validation-playbooks/blacklisted_drivers_validation.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+###
+# This Playbook checks if the some specific drivers are blacklisted in the given image
+# Author: Anuj Maurya  
+###
+---
+
+- name: Print RHEL minor version
```
Comment (simratsatia) [inline ansible_image_validation/validation-playbooks/blacklisted_drivers_validation.yaml:7]: ansible_distribution_version would provide both major and minor version as in "7.9". So, name can be 'RHEL Version'

Comment (anujkmaurya) [reply ansible_image_validation/validation-playbooks/blacklisted_drivers_validation.yaml:7]: Add minor version comments specifically to help developers understand what we are looking for in this task.
We have another variable to check RHEL minor version in Ansible

File: ansible_image_validation/validate-vm-images.yaml (hunk 1 above)
Comment (anujkmaurya) [reply ansible_image_validation/validate-vm-images.yaml]: Done

**Review by simratsatia — APPROVED** [review]

**Review by manikroyin — APPROVED** [review]

**Pull request merged**